content/get-started/using-git/dealing-with-special-characters-in-branch-and-tag-names.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Dealing with special characters in branch and tag names
+intro: 'Git is very permissive about what characters are allowed in branch and tag names. When using Git from a command-line shell, you may need to deal with special characters by using escaping or quoting.'
+versions:
+  fpt: '*'
+  ghes: '*'
+  ghae: '*'
+shortTitle: Special characters in names
+---
+Most repositories use simple branch names like `main` or `update-icons`. Likewise, tag names usually follow a basic format, such as a version number like `v1.2.3`. Both branch and tag names may also use the path separator (`/`) for structure (for example, `area/item`, or `level-1/level-2/level-3`). Apart from a handful of exceptions &mdash; such as not starting or ending a name with a slash, or having consecutive slashes in the name &mdash; Git places [very few restrictions](https://git-scm.com/docs/git-check-ref-format) on what characters may be used.
```

Comment:
Small accessibility suggestion: can we expand the link text from “very few restrictions” to a larger part of the sentence, something like “very few restrictions on what characters may be used in branch and tag names.” Ideally it would make sense in isolation, e.g. when using a screen reader in links mode.

(see [WCAG 2.1, success criterion 2.4.4: Link Purpose (In Context)][2.4.4] and [WCAG 2.1, success criterion 2.4.9: Link Purpose (Link Only)][2.4.9])

[2.4.4]: https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html
[2.4.9]: https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html

Reply:
Good call. Thanks @georgebrock!